### ISSUE TITLE
0.11.X to 0.12.0 Guide Missing

### ISSUE THREAD

**Issue by sshaw** [padrino/padrino-docs#104]
http://www.padrinorb.com/blog/upgrading-padrino-from-0-11-x-to-0-12-0-guide results in a 404; it existed [on the old site](http://web.archive.org/web/20150626104003/http://www.padrinorb.com/blog/upgrading-padrino-from-0-11-x-to-0-12-0-guide)

**Comment by nesquena** [comment]
@wikimatze yeah this does appear to me missing under blog posts now, here's the original <https://web.archive.org/web/20150212063614/http://www.padrinorb.com/blog/upgrading-padrino-from-0-11-x-to-0-12-0-guide>

**Comment by wikimatze** [comment]
@sshaw I've finally ported over the old post and fix some grammar issues. When it's fine, I'll merge it.
